src/components/dialog/_BaseDialog.tsx

```diff
@@ -60,6 +60,7 @@ export const BaseDialog = ({
   width,
 }: BaseDialogProps) => {
   const { aminoTheme } = useAminoTheme();
+  const mouseDownTarget = useRef<HTMLDivElement | null>(null);
```

Comment:
What do you think about removing the `| null`? I believe Ref's always allow `null` 🤔 

```suggestion
  const mouseDownTarget = useRef<HTMLDivElement>(null);
```